docs/usage/self-hosted-configuration.md

```diff
@@ -1093,6 +1110,18 @@ The combinations of `requireConfig` and `onboarding` are:
 | `requireConfig=optional` | An onboarding PR will be created if no config file exists. If the onboarding PR is closed and there's no config file, the repository will be processed. | Repository is processed regardless of config file presence.   |
 | `requireConfig=ignored`  | No onboarding PR will be created and repo will be processed while ignoring any config file present.                                                     | Repository is processed, any config file is ignored.          |
 
+## s3Endpoint
+
+If set, Renovate will use this string as the `endpoint` when instantiating the AWS S3 client.
+
+## s3PathStyle
+
+If set, Renovate will enable `forcePathStyle` when instantiating the AWS S3 client.
+
+> Whether to force path style URLs for S3 objects (e.g., `https://s3.amazonaws.com//` instead of `https://.s3.amazonaws.com/`)
```

Comment:
```suggestion
> Whether to force path-style URLs for S3 objects (e.g., `https://s3.amazonaws.com//` instead of `https://.s3.amazonaws.com/`)
```

It feels like we need ` -` here, but this is a direct quote, I think?